Link to each component's README#dependencies (#997)
k8ssandra/k8ssandra: docs/content/en/third-party/_index.md

The K8ssandra software is licensed under the open-source [**Apache-2.0 license agreement**](https://github.com/k8ssandra/k8ssandra/blob/main/LICENSE).

> For each component deployed by K8ssandra, see the following 3rd-party license reports. Each one was generated by the Fossa app. Start in the #dependencies section of each component's GitHub-hosted README. From there, access the latest generated "open source report" link.

Reviewer: Link to Fossa?

Author: Good idea.

> For each component deployed by K8ssandra, see the following 3rd-party license reports. Each one was generated by the Fossa app. Start in the #dependencies section of each component's GitHub-hosted README. From there, access the latest generated "open source report" link.

Reviewer: ```suggestion
For each component deployed by K8ssandra, see the following 3rd-party license reports. Each one was generated by [Fossa](https://fossa.com/). See the "Dependencies" section of each component's GitHub README. From there, access the latest generated "open source report" link.
```

* [K8ssandra dependencies]({{< relref "https://github.com/k8ssandra/k8ssandra#dependencies" >}}) 
* [Cass Operator dependencies]({{< relref "https://github.com/k8ssandra/cass-operator/#dependencies" >}})
* [Medusa Operator dependencies]({{< relref "https://github.com/k8ssandra/medusa-operator/#dependencies" >}})
* [Reaper Operator dependencies]({{< relref "https://github.com/k8ssandra/reaper-operator/#dependencies" >}})
* [Management API for Apache Cassandra dependencies]({{< relref "https://github.com/k8ssandra/management-api-for-apache-cassandra/#dependencies" >}})
* [Go client library for the Reaper for Apache Cassandra REST API dependencies]({{< relref "https://github.com/k8ssandra/reaper-client-go/#dependencies" >}})
* [Cassandra Reaper dependencies]({{< relref "https://github.com/thelastpickle/cassandra-reaper/#dependencies" >}})

> * [Cassandra Medusa dependencies]({{< relref "https://github.com/thelastpickle/cassandra-medusa/#dependencies" >}})

Reviewer: I would remove "dependencies" from each of the links.

Reviewer: ```suggestion
* [K8ssandra]({{< relref "https://github.com/k8ssandra/k8ssandra#dependencies" >}}) 
* [Cass Operator]({{< relref "https://github.com/k8ssandra/cass-operator/#dependencies" >}})
* [Medusa Operator]({{< relref "https://github.com/k8ssandra/medusa-operator/#dependencies" >}})
* [Reaper Operator]({{< relref "https://github.com/k8ssandra/reaper-operator/#dependencies" >}})
* [Management API for Apache Cassandra]({{< relref "https://github.com/k8ssandra/management-api-for-apache-cassandra/#dependencies" >}})
* [Go client library for the Reaper for Apache Cassandra REST API]({{< relref "https://github.com/k8ssandra/reaper-client-go/#dependencies" >}})
* [Cassandra Reaper]({{< relref "https://github.com/thelastpickle/cassandra-reaper/#dependencies" >}})
* [Cassandra Medusa]({{< relref "https://github.com/thelastpickle/cassandra-medusa/#dependencies" >}})
```

Author: Will do.

* [Tasks]({{< relref "tasks" >}}): Need to get something done? Check out the Tasks topics for a helpful collection of outcome-based solutions.
* [Reference]({{< relref "reference" >}}): Explore the K8ssandra configuration interface (Helm charts), the available options, and a Glossary.

> Also, we encourage you to actively participate in the [K8ssandra community](https://k8ssandra.io/community/).

Reviewer: ```suggestion
Join us as part of the [K8ssandra community](https://k8ssandra.io/community/)!
```

Author: Nice.
